lib/portlayer/exec/exec.go

```diff
@@ -108,9 +109,24 @@ func eventCallback(ie events.Event) {
 				StateRunning,
 				StateStopped,
 				StateSuspended:
+
 				log.Debugf("Container(%s) state set to %s via event activity", container.ExecConfig.ID, newState.String())
 				container.State = newState
-				publishContainerEvent(container.ExecConfig.ID, ie.Created(), ie.String())
+				// container state has changed so we need to update the container attributes
+				// we'll do this in a go routine to avoid blocking
+				go func() {
+
```

Comment:
What up with the empty newlines at the beginning and end of this and some of your other, but not all, functions?